<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSNavigationProcessWidget</class>
 <widget class="QWidget" name="QmitkUSNavigationProcessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">
              font-size: 10pt;
              font-weight: bold;
            </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="stepsToolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>378</width>
        <height>300</height>
       </rect>
      </property>
      <attribute name="label">
       <string>No navigation step widgets loaded.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="navigationButtonsLayout">
     <item>
      <widget class="QToolButton" name="settingsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Open Preferences</string>
       </property>
       <property name="text">
        <string>Preferences</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/preferences.png</normaloff>:/USNavigation/preferences.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartStepButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Restart the Current Step</string>
       </property>
       <property name="text">
        <string>&amp;Restart Step</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/restart.png</normaloff>:/USNavigation/restart.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="previousButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Go to Previous Step</string>
       </property>
       <property name="text">
        <string>&amp;Prev</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/go-previous.png</normaloff>:/USNavigation/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Go to Next Step</string>
       </property>
       <property name="text">
        <string>&amp;Next</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/USNavigation.qrc">
         <normaloff>:/USNavigation/go-next.png</normaloff>:/USNavigation/go-next.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="settingsFrameWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="settingsTitleLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="settingsWidget"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="settingsButtonsLayout">
        <item>
         <widget class="QPushButton" name="settingsSaveButton">
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/USNavigation.qrc">
            <normaloff>:/USNavigation/document-save.png</normaloff>:/USNavigation/document-save.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsCancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/USNavigation.qrc">
            <normaloff>:/USNavigation/restart.png</normaloff>:/USNavigation/restart.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/USNavigation.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnNextButtonClicked()</slot>
  <slot>OnPreviousButtonClicked()</slot>
  <slot>OnRestartStepButtonClicked()</slot>
  <slot>OnTabChanged(int)</slot>
 </slots>
</ui>
